patches/tools-licenses.py.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/tools/licenses.py b/tools/licenses.py
+index 791efb55b47141db7b961784f80f1dedd57e8823..4ee9eeee05c5de48f3488583717a45af02392a50 100755
+--- a/tools/licenses.py
++++ b/tools/licenses.py
+@@ -31,6 +31,8 @@ import tempfile
+ _REPOSITORY_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+ sys.path.insert(0, os.path.join(_REPOSITORY_ROOT, 'build/android/gyp'))
+ from util import build_utils
++sys.path.insert(0, os.path.join(_REPOSITORY_ROOT, 'brave', 'script'))
```

Comment:
`brave/script` is already added to the python path in config.js

Reply:
If I don't add it to the path, I get a build error:
```
[1/155] ACTION //components/resources:about_credits(//build/toolchain/linux:clang_x64)
FAILED: gen/components/resources/about_credits.html 
python ../../tools/licenses.py --target-os=linux --depfile gen/components/resources/about_credits.d credits gen/components/resources/about_credits.html
Traceback (most recent call last):
  File "../../tools/licenses.py", line 34, in <module>
    import licenses
  File "/home/francois/devel/brave-browser/src/tools/licenses.py", line 294, in <module>
    (PRUNE_DIRS, ADDITIONAL_PATHS) = licenses.BravePatch(PRUNE_PATHS, SPECIAL_CASES, PRUNE_DIRS, ADDITIONAL_PATHS)
AttributeError: 'module' object has no attribute 'BravePatch'
```

Reply:
I believe this is because you're importing a module with the same name and we append our python path. We may want to change that so we can override chromium imports, but for now I think just change the name to something else

Reply:
Indeed that was the problem. After renaming, I was able to take that line out.